test/test_build.py

```diff
@@ -53,6 +53,8 @@ def run(self):
 class RunTest(unittest.TestCase):
 
     def setUp(self):
+        luigi.setup_logging.DaemonLogging._configured = False
+        luigi.setup_logging.InterfaceLogging._configured = False
```

Comment:
The same way as testing of luigi
https://github.com/spotify/luigi/blob/4d0576c7c265afcb228097af79f316ba0de0242c/test/cmdline_test.py#L110